test/test_namedtensor.py

```diff
@@ -275,12 +275,11 @@ def test_no_multiprocessing_support(self):
         with self.assertRaisesRegex(RuntimeError, "NYI"):
             ForkingPickler(buf, pickle.HIGHEST_PROTOCOL).dump(named_tensor)
 
-    @unittest.skip("Issue 27753")
-    def test_big_tensor_repr(self):
+    def test_big_tensor_repr_has_names(self):
         def check_repr(named_tensor):
             unnamed_tensor = named_tensor.rename(None)
-            expected = "{}, names={})".format(repr(unnamed_tensor)[:-1], named_tensor.names)
-            self.assertEqual(repr(named_tensor), expected)
+            names_tag = 'names={}'.format(named_tensor.names)
+            self.assertTrue(names_tag in repr(named_tensor))
```

Comment:
nit: we could use assertRegexpMatches so that any error message would print both of the non-matching strings

Reply:
I'm going to use `self.assertEquals`

Reply:
It turns out there is something called `assertIn` that solves this problem